client/src/AppBundle/Resources/views/Ndr/Asset/_list.html.twig

```diff
@@ -20,7 +20,7 @@
         {{ 'existPage.form.noAssetToAdd.label' | trans(transOptions)  }}
     </dt>
     <dd class="govuk-summary-list__value">
-        {{ ndr.noAssetToAdd ? 'Yes' : 'No' }}
+        {{ ndr.noAssetToAdd ? 'No' : 'Yes' }}
```

Comment:
is this reverted back to how it was before 2714 was introduced? If not then we'll have the same issue with existing data when released to production

Reply:
Yep, this is a reversal so it will work the same as before 2714.

Reply:
cool